ucm2/chtmax98090/HiFi-Quawks.conf

```diff
@@ -0,0 +1,18 @@
+SectionVerb {
+	Value {
+		TQ "HiFi"
+	}
+
+	EnableSequence [
+		<platforms/bytcr/PlatformEnableSeq.conf>
```

Comment:
if you ever want to use SOF on this device this would need to change, see https://github.com/alsa-project/alsa-ucm-conf/pull/20

Also I quickly checked the values, they all seem to match what we used so we should just add a default chtmax98090/chtmax98090.conf that calls a basic HiFi.conf and this would work fine on Cyan as well.

Reply:
I added the condition to b081ac49ad9cbea298c3dcc6f029f331e369a213 , but the SOF/HiFi.conf should be modified for this platform, too.

Any generic cleanups would be fine, of course.